eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```diff
@@ -83,6 +85,18 @@ services:
         tags:
             - { name: liip_imagine.cache.resolver, resolver: ezpublish }
 
+    ezpublish.image_alias.imagine.cache_resolver_decorator_factory:
+        class: '%ezpublish.image_alias.imagine.decorated_cache_resolver_factory.class%'
+        arguments:
+            - '@ezpublish.config.resolver.chain'
```

Comment:
Any specific reason for `.chain` here? Usually we use `@ezpublish.config.resolver`.

Reply:
Actually not, wasn't sure which one should I use :) Do you want me to change it to `@ezpublish.config.resolver`?